packages/insured-bridge-relayer/src/canonical-bridge-adapters/OptimismBridgeAdapter.ts

```diff
@@ -0,0 +1,90 @@
+import winston from "winston";
+import { ZERO_ADDRESS } from "@uma/common";
+
+import { providers } from "ethers";
+
+import { getMessagesAndProofsForL2Transaction } from "@eth-optimism/message-relayer";
+import { predeploys, getContractDefinition } from "@eth-optimism/contracts";
+import { getDeployedContractArtifact } from "@eth-optimism/contracts/dist/contract-deployed-artifacts";
+
+import BridgeAdapterInterface from "./BridgeAdapterInterface";
+
+import Web3 from "web3";
+import type { TransactionType } from "@uma/common";
+
+export class OptimismBridgeAdapter implements BridgeAdapterInterface {
+  private l1EthersProvider: any;
+  private l2EthersProvider: any;
```

Comment:
Don't we have well-defined types for Ethers providers that we could use here?

Reply:
ye, sorry. I was lazy with this. have fixed